packages/hardware/quirks/devices/AYN Odin 2/bin/battery_led_status

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024 ROCKNIX (https://github.com/ROCKNIX)
+
+# Simple script to watch the battery capacity and
+# turn the power LED orange when it reaches 30%, red at 20%, and blinking red at 10%
+
+. /etc/profile
+
+LED_PATH="/sys/class/leds"
+
+function led_brightness() {
+  echo ${2} >${LED_PATH}/${1}/brightness
+}
+
+function led_rgb() {
+  echo ${2} ${3} ${4} >${LED_PATH}/${1}/multi_intensity
+}
+
+function bat_led_off() {
+  led_brightness left-side 0
+  led_rgb left-side 0 0 0
+  led_brightness right-side 0
+  led_rgb right-side 0 0 0
+}
+
+function bat_led_red() {
+  led_brightness left-side 255
+  led_rgb left-side 255 0 0
+  led_brightness right-side 255
+  led_rgb right-side 255 0 0
+}
+
+function bat_led_orange() {
+  led_brightness left-side 127
+  led_rgb left-side 255 20 0
+  led_brightness right-side 127
+  led_rgb right-side 255 20 0
+}
+
+PREV_BATCAP="null"
+while true
+  do
+  BAT_LED_STATE=$(get_setting led.color)
+  if [ ! ${BAT_LED_STATE} == "battery" ]; then
+    break
+  else
```

Comment:
You could replace the else with fi to keep the code semantically equivalent while reducing the indentation below.